HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+0.12.1
```

Comment:
Can we please plug this change set under `0.12.2`?

Reply:
sure thing